base/README.md

```diff
@@ -0,0 +1,12 @@
+This creates a patch to `zipkin-anormdb` that avoids HikariDB until it doesn't use `javax.beans` package.
```

Comment:
I'd maybe add some comment here to clarify that "this" is what comes after, as opposed to the whole of `base`. But it's just temporary, so meh.